Resources/Prototypes/_SpaceCats/Roles/Jobs/Security/SecEng.yml

```
@@ -0,0 +1,47 @@
+    back: ClothingBackpackSecurityEngineer
+    shoes: ClothingShoesBootsJackSecEnginer
+    eyes: ClothingEyesHudSecEng
+    gloves: ClothingHandsGlovesColorYellow
+    head: ClothingHeadHatHardhatYellow
+    id: SecurityEngineerPDA
+    outerClothing: ClothingOuterVestHazard
+    ears: ClothingHeadsetSecurityEngineer
+    belt: ClothingMilitaryWebbingSecEng
+  storage:
+    back:
+    - SheetSteel10
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)